utils/errorHandler.ts

```diff
@@ -26,3 +26,6 @@ export const genericInternalValidationErrorHandler = (
   context.log.error(logMessage, errorsToReadableMessages(error).join(" / "));
   return ResponseErrorInternal(errorDetail);
 };
+
+export const errorsToError = (errors: Errors): Error =>
+  new Error(errorsToReadableMessages(errors).join(" / "));
```

Comment:
can we use `readableReportSimplified` method here instead of the old one? @gquadrati 
```suggestion
  new Error(readableReportSimplified(errors));
```